src/middleware/onlyAuthenticated.js

```diff
@@ -0,0 +1,15 @@
+require('dotenv').config();
+const jwt = require('jsonwebtoken');
+
+module.exports = function onlyAuthenticated(req, res, next) {
+  const { accessToken } = req.cookies;
+  try {
+    const error = { meesage: 'There is no session; please sign in.' };
+    if (!accessToken) throw error;
+    const user = jwt.verify(accessToken, process.env.SECRET_KEY);
+    req.user = user;
```

Comment:
I wonder if it is possible to write the condition if (req.user) here?
so there is no need to write it in each function separately